docs/configuration.md

```diff
@@ -521,15 +521,21 @@ Ruff: An extremely fast Python linter.
 Usage: ruff [OPTIONS] <COMMAND>
 
 Commands:
-  check    Run Ruff on the given files or directories (default)
-  rule     Explain a rule (or all rules)
-  config   List or describe the available configuration options
-  linter   List all supported upstream linters
-  clean    Clear any caches in the current directory and any subdirectories
-  format   Run the Ruff formatter on the given files or directories
-  server   Run the language server
-  version  Display Ruff's version
-  help     Print this message or the help of the given subcommand(s)
+  check                      Run Ruff on the given files or directories
+                                 (default)
+  rule                       Explain a rule (or all rules)
+  config                     List or describe the available configuration
+                                 options
+  linter                     List all supported upstream linters
+  clean                      Clear any caches in the current directory and
+                                 any subdirectories
+  generate-shell-completion  Generate shell completion
```

Comment:
I feel like we've kept this hidden previously? Definitely nicer indentation when hidden.

Reply:
Nice catch. Yeah, I deleted a little too much. Thank you